waspc/src/Wasp/Util/Terminal.hs

```diff
@@ -39,6 +43,7 @@ styleCode Cyan = "[36m"
 styleCode White = "[37m"
 styleCode Bold = "[1m"
 styleCode Underline = "[4m"
+styleCode Blink = "[5m"
```

Comment:
This might not work in some terminal emulators 😢

Reply:
I added a comment about that.